src/mrsimulator/methods/tests/test_method_updates.py

```diff
@@ -66,7 +66,8 @@ def assert_parsing(method, fn1):
             method(**serialize)
 
         with pytest.raises(ImmutableEventError, match=f".*{event_error}.*"):
-            ent.append({"transition_query": [{"ch1": {"P": [-100]}}]})
+            ent[0]["fraction"] = 0.5
```

Comment:
Why do you have to set this to 0.5?

Reply:
Was due to raising ``ValidatonError`` when the sum of fractions became 2 after adding a second event. After implementing warning, change can either be reverted or kept.